openapi-core/src/main/java/io/ballerina/openapi/core/generators/document/TypesDocCommentGenerator.java

```diff
@@ -62,10 +62,13 @@ public SyntaxTree updateSyntaxTreeWithDocComments() {
         Map<String, Schema> schemas = openAPI.getComponents().getSchemas();
         ModulePartNode modulePartNode = (ModulePartNode) rootNode;
         NodeList<ModuleMemberDeclarationNode> members = modulePartNode.members();
-        List<TypeDefinitionNode> updatedList = new ArrayList<>();
+        List<ModuleMemberDeclarationNode> updatedList = new ArrayList<>();
```

Comment:
Seems like we are collecting the non-type member nodes to this list, can we rename this to reflect that purpose?

Reply:
not only that. we add the type def nodes to this as well. will rename to `updatedMemberTypesList`

Reply:
updated